src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -749,29 +719,10 @@ messages.stmsg.m0022 = Submit one of your drafts for review. Content is publishe
 messages.stmsg.m0024 = Upload a document, video, or any other supported format. You have not uploaded anything yet
 messages.stmsg.m0033 = Submit one of your drafts for review. You have not yet submitted any content for review
 messages.stmsg.m0035 = There is no content to review
-messages.stmsg.m0060 = Strengthen your profile
-messages.stmsg.m0062 = Enter valid degree
-messages.stmsg.m0063 = Enter valid Address line 1
-messages.stmsg.m0064 = Enter City
-messages.stmsg.m0065 = Enter valid pin code
-messages.stmsg.m0066 = Enter First name
-messages.stmsg.m0067 = Provide a valid mobile number
-messages.stmsg.m0069 = Select language
-messages.stmsg.m0070 = Enter Instituition name
-messages.stmsg.m0072 = Enter valid Occupation / Work Title
-messages.stmsg.m0073 = Enter valid Organization
 messages.stmsg.m0077 = We are submitting your request...
 messages.stmsg.m0081 = No batches found
 messages.stmsg.m0083 = You have not shared content with any one yet
-messages.stmsg.m0088 = Enter a valid password
-messages.stmsg.m0089 = Enter a valid email address
-messages.stmsg.m0090 = Select languages
-messages.stmsg.m0091 = Enter a valid mobile number
-messages.stmsg.m0094 = Enter valid percentage
-messages.stmsg.m0095 = Your request is received successfully. The file will be sent to your registered email address shortly. Check your registered email address to view it.
-messages.stmsg.m0104 = Enter valid grade
 messages.stmsg.m0108 = Your Progress
-messages.stmsg.m0113 = Enter a valid start date
```

Comment:
this is used in multiple html file: messages.stmsg.m0113 = Enter a valid start date
check create-course-batch, update-course-batch, create-batch component's HTML file